travis/build-wheels.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e -x
+
+cd /io
+mkdir -p deps
+cd deps
+
+yum install -y git cmake yum wget
+wget http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.2.tar.gz
+tar -xvf ruby-2.1.2.tar.gz
+cd ruby-2.1.2
+./configure
+make -j4
+make install
+cd ..
+rm -rf ruby-2.1.2
+
+git clone --branch v2.2.12 https://github.com/OCL-dev/ocl-icd
+cd ocl-icd
+wget https://raw.githubusercontent.com/conda-forge/ocl-icd-feedstock/master/recipe/install-headers.patch --no-check-certificate
+git apply install-headers.patch
+autoreconf -i
+chmod +x configure
+./configure --prefix=/usr
+make -j4
+make install
+
+cd /io
+
+# Compile wheels
+for PYBIN in /opt/python/*/bin; do
+    "${PYBIN}/pip" install numpy pybind11 mako
+    "${PYBIN}/pip" wheel /io/ -w wheelhouse/
+done
```

Comment:
Can you change the above 4 lines to,

```bash
for PYBIN in /opt/python/*/bin; do
    if [[ "${PYBIN}" == *cp36* ]]; then
        NUMPY_VERSION="1.11.3"
    elif [[ "${PYBIN}" == *cp37* ]]; then
        NUMPY_VERSION="1.14.5"
    elif [[ "${PYBIN}" == *cp35* ]]; then
        NUMPY_VERSION="1.9.3"
    else
        NUMPY_VERSION="1.8.2"
    fi
    # Build with the oldest numpy available to be compatible with newer ones
    "${PYBIN}/pip" install "numpy==${NUMPY_VERSION}" pybind11 mako
    "${PYBIN}/pip" wheel /io/ -w wheelhouse/
done
```

Reply:
Or I can send a newer PR. This is the only thing I needed to change in the last PR.

Reply:
Like this?

Reply:
yes, thanks. let me run this on a docker container and confirm.

Reply:
Needed a few other changes and added in https://github.com/inducer/pyopencl/pull/273